File: silver-logs/src/dtos/ReportBadgeDto.js
export default class ReportBadgeDto{
  /**
   * @param {LocalDate} weekStart
   * @param {LocalDate} weekEnd
   * @param {number} reportNumber
   * @param {boolean} submitted
   * @param {boolean} approved
   * @param {boolean} rejected
   */
  constructor(weekStart, weekEnd,reportNumber, submitted, approved, rejected) {
    this.weekStart = weekStart;
    this.weekEnd = weekEnd;
    this.reportNumber = reportNumber
    this.submitted = submitted;
    this.approved = approved;
    this.rejected = rejected;
  }
};

File: SilverLogsBackend/src/main/java/com/SilverGGR/SilverLogs/dtos/ReportBadgeDto.java
package com.SilverGGR.SilverLogs.dtos;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDate;

@Getter
@Setter
public class ReportBadgeDto {
    private LocalDate weekStart;
    private LocalDate weekEnd;
    private Integer reportNumber;
    private Boolean submitted;
    private Boolean approved;
    private Boolean rejected;
}
